modules/local/assemblystub/main.nf

```diff
@@ -2,7 +2,9 @@ process ASSEMBLY_STUB {
     tag "$meta.id"
     label 'process_single'
 
-    container 'docker.io/python:3.9.17'
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.10' :
+        'biocontainers/python:3.10' }"
```

Comment:
My logic was that when the python version was not available as a biocontainer to go to the next version above. In this case, since it was using 3.9.17 I switched to 3.12, whereas something like arborview was using python:3.11.6, so I went to 3.12. Trying my best to avoid introducing dependency issues.